### DISCUSSION TITLE
Questions related to connection between Codemagic and AWS device farm

### DISCUSSION THREAD

**Discussion by nkuzmanovic-litebit** [codemagic-ci-cd/codemagic-docs#1324] [Q&A] [answered]
1. I'm interested if it is possible to get test results from AWS after running integration tests or E2E tests (Patrol) on its device farm, and to have that data here in Codemagic?
If that is possible, one example of how it can be done is nice to have :D 

2. Example or idea how to point to specific device when running integration tests against AWS device farm from Codemagic?

**Comment by icarusdust** [comment, marked as answer]
Hi @nkuzmanovic-litebit, I think we have already discussed it in the other channel ;)

**Reply by nkuzmanovic-litebit** [reply]
Yes, thank you :)
